pointerevents/pointerevent_attributes_mouse-manual.html

```diff
@@ -17,68 +17,87 @@
             // will fail unless the async_test is created with the var name "test_pointerEvent".
             add_completion_callback(showPointerTypes);
 
+            function checkPointerEventAttributes(event, targetBoundingClientRect, testNamePrefix, pointerEventInterface) {
+                if (detected_eventTypes[event.type])
+                    return;
+                var pointerTestName = testNamePrefix + ' ' + event.pointerType + ' ' + event.type;
+                test(function () {
+                    assert_equals(event.pointerType, 'mouse', 'pointerType should be mouse');
+                }, pointerTestName + ".pointerType attribute is correct.");
+
+                detected_pointertypes[event.pointerType] = true;
+                // Test button and buttons
+                if (event.type == 'pointerdown') {
+                    test(function() {
+                        assert_true(event.button == 0, "If left mouse button is pressed button attribute is 0")
+                    }, pointerTestName + "'s button attribute is 0 when left mouse button is pressed.");
+                    test(function() {
+                        assert_true(event.buttons == 1, "If left mouse button is pressed buttons attribute is 1")
+                    }, pointerTestName + "'s buttons attribute is 1 when left mouse button is pressed.");
+                } else if (event.type == 'pointerup') {
+                    test(function() {
+                        assert_true(event.button == 0, "If left mouse button is just released button attribute is 0")
+                    }, pointerTestName + "'s button attribute is 0 when left mouse button is just released.");
+                    test(function() {
+                        assert_true(event.buttons == 0, "If left mouse button is just released buttons attribute is 0")
+                    }, pointerTestName + "'s buttons attribute is 0 when left mouse button is just released.");
+                } else {
+                    test(function() {
+                        assert_true(event.button == -1, "If mouse buttons are released button attribute is -1")
+                    }, pointerTestName + "'s button is -1 when mouse buttons are released.");
+                    test(function() {
+                        assert_true(event.buttons == 0, "If mouse buttons are released buttons attribute is 0")
+                    }, pointerTestName + "'s buttons is 0 when mouse buttons are released.");
+                }
+
+                // Test clientX and clientY
+                if (event.type != 'pointerout' && event.type != 'pointerleave' ) {
+                    test(function () {
+                        assert_true(event.clientX >= targetBoundingClientRect.left && event.clientX < targetBoundingClientRect.right, "ClientX should be in the boundaries of the black box");
```

Comment:
I combined the clientX/Y tests into one as they are related and there was only one test anyway for the pointerout/leave case.